spec/spec_helper.rb

```diff
@@ -0,0 +1,12 @@
+RSpec.configure do |config|
+  config.expect_with :rspec do |expectations|
+    expectations.include_chain_clauses_in_custom_matcher_descriptions = true
+  end
+
+  config.mock_with :rspec do |mocks|
+    mocks.verify_partial_doubles = true
+  end
+
+  config.shared_context_metadata_behavior = :apply_to_host_groups
+  config.order = :random
+end
```

Comment:
I would add

  config.expect_with :rspec do |c|
    c.syntax = :expect
  end

Reply:
yep I thought I had added that one ! my bad